src/test/ui/conditional-compilation/cfg-attr-multi-invalid-2.rs

```diff
@@ -1,7 +1,6 @@
 //
 // compile-flags: --cfg broken
 
-#![feature(cfg_attr_multi)]
```

Comment:
Since this test is meant to fail, maybe you need to `--bless` it? (and possibly the `-invalid-1.rs` file as well)